README.md

```diff
@@ -12,7 +12,7 @@ Python SDK to ensure excellent user experience for developers and data scientist
 
 ## Prerequisites
 In order to start using the Python SDK, you need
-- Python3 and pip
+- Python3 (>= 3.3) and pip
```

Comment:
Could you add `python_requires='>=3.3'` to setup.py?

Reply:
Sure!

Reply:
Done!